### PR TITLE
Adding recipe for Blosc2 3.0.0rc3

### PR DESCRIPTION
**Pull request by FrancescAlted**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/5eddbd7fc9d1502169089da06c3688d9759be978/recipes/example/meta.yaml#L64-L73) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Package does not ship static libraries. If static libraries are needed, [follow CFEP-18](https://github.com/conda-forge/cfep/blob/main/cfep-18.md).
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Review by hmaarrfk — CHANGES_REQUESTED** [review]
This should not be its own recipe, rather please make a pull request to https://github.com/conda-forge/python-blosc2-feedstock

**Comment by hmaarrfk** [comment]
converting to draft to avoid accidental merging.

File: recipes/blosc2/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+build:
+  number: 0
+  skip: true  # [py<311]
+  script_env:
+    - USE_SYSTEM_BLOSC2=1
+  script: |
+    {% if win %}
+    git clone https://github.com/microsoft/vcpkg.git
+    .\vcpkg\bootstrap-vcpkg.bat
+    .\vcpkg\vcpkg install pkg-config
```
Comment (hmaarrfk) [inline recipes/blosc2/meta.yaml:22]: in short, vcpkg is microsoft's answer to conda.

A different strategy. 

At conda-forge, you must build everything (well most things) with conda-forge's stack.

**Comment by FrancescAlted** [comment]
> This should not be its own recipe, rather please make a pull request to https://github.com/conda-forge/python-blosc2-feedstock

Ah! I forgot about this. I'll try this tomorrow.

**Comment by FrancescAlted** [comment]
Should I close this one then?

**Comment by hmaarrfk** [comment]
> Should I close this one then?

yes, but me closing it seems "agressive" so I wanted to ensure some discussion before doing it.

**Comment by hmaarrfk** [comment]
> Ah! I forgot about this. I'll try this tomorrow.

I often use https://conda-forge.org/packages/ to find packages by "name"

**Comment by FrancescAlted** [comment]
Closing this in favor of https://github.com/conda-forge/python-blosc2-feedstock/pull/18

**Pull request closed without merging**